### DISCUSSION TITLE
Sticky filters

### DISCUSSION THREAD

**Discussion by arichard42** [netbox-community/netbox#9611] [Ideas]
I would like to see sticky filters, i.e. filters that are kept when you click on several choices on the left pane.

For example I would like to filter on a tenant, and then click on racks, Prefixes, VRFs, Virtual Machines to see only items relevant to this tenant.

A plus would be that any items created would take the filter as default values. For example if I filter on 'Tenant 1', then click on Add IP Addresses, the Tenant field would be pre-selected for the 'Tenant 1' tenant. 

This will make the whole interface more intuitive and faster.

**Comment by jeremystretch** [comment]
I believe this has been captured in FR #9623.
